src/test/regress/sql/cont_grouping_sets.sql

```diff
@@ -0,0 +1,37 @@
+CREATE CONTINUOUS VIEW test_gs0 AS SELECT x::integer, y::integer, COUNT(*) FROM test_gs_stream
```

Comment:
Maybe also add `SELECT pipeline_get_worker_querydef` (and the ones for combiner and overlay view) to make sure the right plans are being generated.